fdbclient/NativeAPI.actor.cpp

```diff
@@ -2711,7 +2711,7 @@ TransactionOptions::TransactionOptions(Database const& cx) {
 }
 
 TransactionOptions::TransactionOptions() {
-	memset(this, 0, sizeof(*this));
+	memset(static_cast<void*>(this), 0, sizeof(*this));
```

Comment:
I don't _think_ this is problematic, but it seems kind of weird now that there's std::sets and other non-trivial objects as members of TransactionOptions. If UBSAN is happy with it then I think it's probably fine

Reply:
Good point, UBSAN doesn't complain about this, but it does leak memory because we're not freeing the memory from the `std::set`. I'll add a new commit to fix this.